modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/handler/SlashApiHandler.java

```diff
@@ -55,11 +55,16 @@ public class SlashApiHandler
     private static final Pattern MOUNT_SITE_ENDPOINT_PATTERN =
         Pattern.compile( "^/(admin/)?site/(?<project>[^/]+)/(?<branch>[^/]+)(?<contentPath>/.*?)/_/" );
```

Comment:
How about project path? I'm not sure it is captured by this regex

Reply:
We do not have requirements to support project path at least for now.